data/themes/darktable.css

```diff
@@ -723,7 +750,7 @@ spinbutton>button
 }
 
 /* set background color and color of scope/histogram buttons */
-#main-histogram #dt-button
+#main-histogram .dt_module_btn
```

Comment:
This break histogram button and especially hover effect. Just check them on histogram and see hover effect on master and on your PR.

Reply:
Ignore that comment